Core/src/org/sleuthkit/autopsy/corecomponents/Installer.java

```diff
@@ -111,4 +114,18 @@ private void setOSXLookAndFeel() {
             UIManager.put(entry.getKey(), entry.getValue());
         });
     }
+    
+    private void setUnixLookAndFeel(){
+        try {
+            UIManager.setLookAndFeel(UIManager.getCrossPlatformLookAndFeelClassName());
+        } catch (ClassNotFoundException | InstantiationException | IllegalAccessException | UnsupportedLookAndFeelException ex) {
+            logger.log(Level.WARNING, "Error setting crossplatform look-and-feel, setting default look-and-feel",ex);
+            try {
+                UIManager.setLookAndFeel(UIManager.getSystemLookAndFeelClassName());
+                enableTimeline=false;
+            } catch (ClassNotFoundException | InstantiationException | IllegalAccessException | UnsupportedLookAndFeelException ex1) {
+                 logger.log(Level.WARNING, "Error setting native look-and-feel",ex1);
+            }
```

Comment:
Given that you say that the default system LAF will be used anyway, I think we should reduce code complexity here (i.e., eliminate the nested try-catch) and simply set the property to false after logging the error.